app/Resources/views/themes/tastefulplanet/fields/ezrichtext/custom_tags/ezyoutube.html.twig

```diff
@@ -0,0 +1,9 @@
+<div{% if params.align is defined %} style="text-align: {{ params.align }};"{% endif %}>
```

Comment:
`params.align`: Is there a way to replace this value by the value of the alignment buttons?
![image](https://user-images.githubusercontent.com/1157233/37620606-1f3ecd72-2b93-11e8-980d-c286c969038c.png)

Reply:
ping @dew326

Reply:
If it's too complicated for this release, we should remove the buttons and not confuse users.

Reply:
@SylvainGuittard I've established with @dew326 that a proper value of alignment is set and saved to a database, but RichText Custom Tag renderer doesn't pass it to a template, so it's a Kernel bug :scream: 

Reported as [EZP-28972](https://jira.ez.no/browse/EZP-28972).

Reply:
There are not bugs in Kernel, it's only a challenge.

Reply:
![not-bug-its-feature](https://user-images.githubusercontent.com/1736658/37643814-0f0361d6-2c22-11e8-819c-55f44eabb8b3.jpg)

Reply:
Can we have this issue ([EZP-28972](https://jira.ez.no/browse/EZP-28972)) fixed for v2.1.0?

Reply:
Yes, working on it.